src/charm.py

```diff
@@ -275,6 +275,10 @@ def _get_runner_manager(
         else:
             path = GitHubOrg(org=path, group=self.config["group"])
 
+        dockerhub_mirror = (
+            self.config["dockerhub-mirror"] if self.config["dockerhub-mirror"] else None
+        )
+
```

Comment:
This might be a tad bit simpler :D
```suggestion
        dockerhub_mirror = self.config["dockerhub-mirror"] or None
```

```python3
>>> type("" or None)
<class 'NoneType'>
```